FlightReservation/components/modalComponent.js

```diff
@@ -0,0 +1,93 @@
+import React, { useState } from 'react';
+import {
+    StyleSheet,
+    View,
+    Text,
+    Image,
+    Modal,
+  } from 'react-native';
+import { AnimatedCircularProgress } from 'react-native-circular-progress';
+
+function ModalCustom(props) {
+    const { modalVisible, text, isIconCheck } = props;
+    return (
+      <View style={styles.centeredView}>
+        <Modal
+          animationType="slide"
+          transparent={true}
+          visible={modalVisible}
+          presentationStyle="overFullScreen"
+        >
+          <View style={[styles.centeredView, {backgroundColor:'rgba(0, 0, 0, 0.4)' }]}>
+            <View style={[styles.modalView, {backgroundColor: "black"}]}>
+              {isIconCheck
+                ? <AnimatedCircularProgress
+                  size={80}
+                  width={8}
+                  fill={100}
+                  tintColor="#5362D8"
+                  backgroundColor="#3d5875"
+                  duration={2500}
+                />
+                : <Image
+                  style={styles.iconCheck}
+                  source={require('../img/check.png')}
+                />
+              }
+              <Text style={styles.modalText}>{text}</Text>
+            </View>
+          </View>
+        </Modal>
+      </View>
+    );
+  }
+  
+  const styles = StyleSheet.create({
+      //Estilos de modal
+  centeredView: {
+    flex: 1,
+    justifyContent: "center",
+    alignItems: "center",
+    marginTop: 22,
+  },
+  modalView: {
+    margin: 10,
+    backgroundColor: "white",
+    borderRadius: 10,
+    padding: 35,
+    paddingBottom: 10,
+    alignItems: "center",
+    shadowColor: "#000",
```

Comment:
same thing for naming colors
I have a super bad memory and I don't remember if this is ⚪  or ⚫